src/plugins/data-card/components/sort-card-attribute.tsx

```diff
@@ -0,0 +1,43 @@
+import React, { useState } from "react";
+import { ITileModel } from "../../../models/tiles/tile-model";
+import { DataCardContentModelType } from "../data-card-content";
+import { gImageMap } from "../../../models/image-map";
+import { IAttribute } from "../../../models/data/attribute";
+
+interface IProps {
+  caseId: string;
+  model: ITileModel;
+  attr: IAttribute;
+}
+
+const getTruncated = (val:string) => {
+  if (!val) return;
+
+  const charTarget = 48;
+  if (val.length < charTarget + 1) return val;
+
+  const firstRelevantSpace = val.indexOf(" ", charTarget) + 1;
```

Comment:
I think `firstRelevantSpace` will be 0 if there is no space after character 48 here, which will make this function return `"..."`, which seems wrong. I saw you used a custom truncating function elsewhere as well... is there a reason css automatic truncation wouldn't work for these cases?

Reply:
Oh, thanks! You are right. I didn't know CSS truncation existed... just looked it up...been around forever.  😮  Anyway this ended up making me realize that in the new world of expandable/collapsable cards, there is not really a reason for truncation of the value in the sort view (in fact, it's the only place you can see it at the moment).  For the category heading, I left that custom function in because we have to have the text value have a flexible width, and not cut off the word "cards", and I thought it might be awkward to have the string span parent/child divs?